contracts/swaps/SwapsUser.sol

```diff
@@ -259,7 +259,8 @@ contract SwapsUser is State, SwapsEvents, FeesHelper {
             sourceToken,
```

Comment:
this call might need to be changed depending on what we decide on using swapsImpl directly (not via delegatecall from the protocol)